var/spack/repos/builtin/packages/py-uproot3/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyUproot3(PythonPackage):
```

Comment:
Explained a bit above, but I think you should use `py-uproot` instead of `py-uproot3`. uproot3 and uproot4 seem to no longer be getting updates on PyPI.

Reply:
Yep, per my comments, uproot3 is needed. It does seem to be getting updates, at least as of February 2021.